README.md

```diff
@@ -50,13 +50,13 @@ Examples
 
 To acquire a single certificate for both `example.com` and `*.example.com`, waiting 900 seconds for DNS propagation:
 
-    certbot certonly \\
-      --authenticator dns-godaddy \\
-      --dns-godaddy-credentials ~/.secrets/certbot/godaddy.ini \\
```

Comment:
This example should remain, even if it is reworded, so that people know how to use a file to provide the credentials.

Reply:
I'm not sure why you're seeing this as removed, in my code it appears as:

    sudo certbot certonly \
      --authenticator dns-godaddy \
      --dns-godaddy-propagation-seconds 900 \      
      --dns-godaddy-credentials /var/lib/letsencrypt/godaddy_credentials.ini \
      --keep-until-expiring --non-interactive --expand \
      --server https://acme-v02.api.letsencrypt.org/directory \
      -d 'example.com' \
      -d '*.example.com'